include/gatekeeper_fib.h

```diff
@@ -0,0 +1,208 @@
+/*
+ * Gatekeeper - DoS protection system.
+ * Copyright (C) 2016 Digirati LTDA.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _GATEKEEPER_GK_FIB_H_
+#define _GATEKEEPER_GK_FIB_H_
+
+#include <rte_ether.h>
+#include <rte_hash.h>
+
+#include "gatekeeper_net.h"
+#include "gatekeeper_lpm.h"
+
+/*
+ * The LPM reserves 24-bit for the next-hop field.
+ * TODO Drop the constant below and make it dynamic.
+ */
+#define GK_MAX_NUM_FIB_ENTRIES (256)
+
+/* TODO Support more fib actions. */
```

Comment:
The previous patch has already added all the needed FIB actions. So, I will drop this comment.